src/clp_ffi_js/ir/StreamReader.cpp

```diff
@@ -1,44 +1,28 @@
 #include "StreamReader.hpp"
 
-#include <algorithm>
 #include <cstddef>
 #include <cstdint>
-#include <iterator>
+#include <ErrorCode.hpp>
```

Comment:
Not sure how the linter is not catching this but we want to rename this import as 
```
#include <clp/ErrorCode.hpp>`
```
and move it to the import group below.

Reply:
We misconfigured the include directories in `CMakeLists.txt` It's because we configured `src/submodules/clp/components/core/src/clp` as an include directory in `CMakeLists.txt`, primarily because we treat `string_utils` as an independent library in clp-core, but it's still under the `clp` dir. I'll open an issue in clp to track and fix this. For now, let's double-check that all clp includes aren't mistakenly included in the standard C++ includes.

Reply:
https://github.com/y-scope/clp/issues/571